bin/mail/2017-02-09-annual-report/send-real-emails

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# 2372749 contacts
+../send-to-mailing-list --start 0 --limit 500000 2017-02-09-annual-report ./contacts.csv
+../send-to-mailing-list --start 500000 --limit 500000 2017-02-09-annual-report ./contacts.csv
+../send-to-mailing-list --start 1000000 --limit 500000 2017-02-09-annual-report ./contacts.csv
+../send-to-mailing-list --start 1500000 --limit 500000 2017-02-09-annual-report ./contacts.csv
+../send-to-mailing-list --start 2000000 --limit 400000 2017-02-09-annual-report ./contacts.csv
```

Comment:
Perhaps. What are you thinking? 

I don't even think we need to break these up. Production-console is pretty beefy, but I still do it to be safe (you know, it's the way it's always been done...) on these large lists.